packages/components/src/FormField/components/ClearAction.css

```diff
@@ -0,0 +1,25 @@
+.clearContainer {
+  display: flex;
+  align-items: center;
+  padding-left: var(--space-small);
+}
+
+.clearInput {
+  display: flex;
+  position: relative;
+  right: var(--space-base);
+  z-index: var(--elevation-tooltip);
+  width: var(--space-large);
+  height: var(--space-large);
+  padding: 0;
+  border: none;
+  border-radius: var(--radius-circle);
+  background-color: var(--color-surface--background);
+  cursor: pointer;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
We need focus states here, can we write them using `:focus-visible` (see [ComboboxClearAction](https://github.com/GetJobber/atlantis/blob/979b53e34b1e16cbbb7de53184c1d9eb42dba8d7/packages/components/src/Combobox/components/ComboboxAction/ComboboxAction.css#L21-L33) for reference)

Reply:
Just need to make sure we're clearing the `outline` property and setting the `box-shadow: var(--shadow-focus)`